### ISSUE TITLE
How to wrap cell text if the row exceeds some width

### ISSUE THREAD

**Issue by numToStr** [tov/tabular-rs#14]
Thank you for making this little crate ❤️ 

So, I am working on a tool that generates some docs but the problem is that the text width that the docs can have is 80 characters. So I was wondering whether the following is possible?

This is how it renders now (no wrapping)

```
Fields: ~
    {number}  (toggle)     Toggle action can be something that can exceeds 80 chars limit of the docs
    {number}  (comment)    Comment action
    {number}  (uncomment)  Uncomment action
```

This is what I want

```
Fields: ~
    {number}  (toggle)     Toggle action can be something that can exceeds 80 chars
                           limit of the docs
    {number}  (comment)    Comment action
    {number}  (uncomment)  Uncomment action
```

I've also tried `with_custom_width_cell` but there is no change, maybe I am using it wrong.

**Comment by sunshowers** [comment]
That isn't currently supported, but it sounds like a reasonable feature request. A PR would be welcome!

This can get tricky if multiple columns need to be wrapped I think, so maybe let's just do the last column.

**Comment by numToStr** [comment]
Yes, the last column wrapping is what i wanted. This is the same as clap which wraps the last column in help for args/options.

**Comment by sunshowers** [comment]
Yeah, definitely sounds useful. Worth looking at https://docs.rs/textwrap if you're going to implement this -- I think you may wish to expose some of the details yourself.

Also worth thinking about text that has embedded newlines in it.
